--- collectd-5.0.0/src/notify_desktop.c.wiget	2011-04-13 18:20:49.328905159 +0200
+++ collectd-5.0.0/src/notify_desktop.c	2011-04-13 18:21:41.933248544 +0200
@@ -95,7 +95,7 @@ static int c_notify (const notification_
 				: (NOTIF_WARNING == n->severity) ? "WARNING"
 				: (NOTIF_OKAY == n->severity) ? "OKAY" : "UNKNOWN");
 
-	notification = notify_notification_new (summary, n->message, NULL, NULL);
+	notification = notify_notification_new (summary, n->message, NULL);
 	if (NULL == notification) {
 		log_err ("Failed to create a new notification.");
 		return -1;
